packages/shared/components/modals/NetworkSummaryModal.svelte

```diff
@@ -15,7 +15,7 @@
 <Modal bind:this={modal} position={{ left: '80px', top: '25px' }}>
     <network-indicator-content class="flex flex-col">
         {#if $activeProfile?.isDeveloperProfile}
-            <Text type="h3" classes="px-7 pt-5">{$activeProfile?.settings.networkConfig.network.name}</Text>
+            <Text type="h3" classes="px-7 pt-5">{$activeProfile?.settings.clientOptions.network}</Text>
```

Comment:
This is never set locally so it is undefined, we probably should get the network from the node info.